packages/ui5-middleware-serveframework/lib/index.js

```diff
@@ -73,16 +74,24 @@ module.exports = async ({ log, options, middlewareUtil }) => {
 
 			// support for coporate proxies
 			const { getProxyForUrl } = await import("proxy-from-env");
-			const proxyUrl = getProxyForUrl(baseUrl);
 			const { HttpsProxyAgent } = await import("https-proxy-agent");
-			const agent = proxyUrl ? new HttpsProxyAgent(proxyUrl) : undefined;
-			effectiveOptions.debug && log.info(`[${baseUrl}] Proxy: ${proxyUrl ? proxyUrl : "n/a"}`);
+			const https = await import("https");
```

Comment:
Native modules should be imported on top of the file with require and you can use object destructuring to directly access the agent:

```js
const { Agent: HttpsAgent } = require("https");
```